test/ping/ping-02-errors.t

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/perl -w
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2025 Petr Vorel <pvorel@suse.cz>
+
+use Test::Command tests => 6;
+use Test::More;
+
+my $lib = File::Basename::dirname(Cwd::abs_path($0)) . '/../lib.pl';
+require "$lib";
+
+my $ping = get_cmd($ARGV[0] // 'ping');
+
+# no arg
+{
+	my $cmd = Test::Command->new(cmd => "$ping", env => { LC_ALL => 'C', LANG   => 'C' });
+	$cmd->exit_is_num(2);
+	subtest 'output' => sub {
+		$cmd->stderr_is_eq("$ping: usage error: Destination address required\n");
+	}
+}
+
+# -c1 -i0.001 127.0.0.1
+{
+	my $cmd = Test::Command->new(cmd => "$ping -c1 -i0.001 127.0.0.1", env => { LC_ALL => 'C', LANG   => 'C' });
```

Comment:
Shouldn't the env with `LC_ALL='C'` be passed in all tests? The `ping-01-basic.t` does not seem to do that. Or we can set these once before all the tests so that we do not need to repeat them in each test that is matching the output.

Reply:
>  I'll put setup into [`test/lib.pl`](https://github.com/pevik/iputils/blob/testing/test/lib.pl)...